cardano-testnet/src/Testnet/SubmitApi.hs

```diff
@@ -0,0 +1,140 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+{-# OPTIONS_GHC -Wno-unused-imports -Wno-unused-local-binds -Wno-unused-matches #-}
+
+module Testnet.SubmitApi
+  ( submitApi
+  , submitApiSubmitTransaction
+  ) where
+
+import           Control.Exception (IOException, handle)
+import           Control.Monad
+import           Control.Monad.IO.Class
+import           Data.Aeson (FromJSON (..), Value, (.=), (.:))
+import           Data.Bool
+import           Data.Eq
+import           Data.Either
+import           Data.Function
+import           Data.Functor
+import           Data.Int
+import           Data.IORef
+import           Data.Maybe
+import           Data.Ord
+import           Data.Semigroup
+import           Data.String
+import           Data.Text (Text)
+import           GHC.Generics
+import           GHC.Num
+import           Hedgehog (MonadTest)
+import           Hedgehog.Extras.Stock.Aeson (rewriteObject)
+import           Hedgehog.Extras.Stock.IO.Network.Sprocket (Sprocket (..))
+import           Hedgehog.Extras.Stock.Time
+import           System.FilePath.Posix ((</>))
+import           System.IO (FilePath, IO)
+import           Text.Show
+
+import qualified Data.Aeson as J
+import qualified Data.ByteString.Base16 as Base16
+import qualified Data.ByteString as BSS
+import qualified Data.ByteString.Lazy as LBS
+import qualified Data.ByteString.Char8 as BS8
+import qualified Data.HashMap.Lazy as HM
+import qualified Data.List as L
+import qualified Data.Text as T
+import qualified Data.Text.Encoding as T
+import qualified Data.Time.Clock as DTC
+import qualified GHC.Stack as GHC
+import qualified Hedgehog as H
+import qualified Hedgehog.Extras.Stock.IO.File as IO
+import qualified Hedgehog.Extras.Stock.IO.Network.Socket as IO
+import qualified Hedgehog.Extras.Stock.IO.Network.Sprocket as IO
+import qualified Hedgehog.Extras.Stock.String as S
+import qualified Hedgehog.Extras.Test.Base as H
+import qualified Hedgehog.Extras.Test.Concurrent as H
+import qualified Hedgehog.Extras.Test.File as H
+import qualified Hedgehog.Extras.Test.Network as H
+import qualified Hedgehog.Extras.Test.Process as H
```

Comment:
Is there any name collision when importing all `Hedgehog.Extra` modules as `HExtra` ?

Reply:
No know name collisions as yet.